app/v2/models/response/retrieveForeignPropertyPeriodSummary/foreignFhlEea/ForeignFhlEeaIncome.scala

```diff
@@ -16,10 +16,12 @@
 
 package v2.models.response.retrieveForeignPropertyPeriodSummary.foreignFhlEea
 
-import play.api.libs.json.{Json, OFormat}
+import play.api.libs.json.{JsPath, Json, Reads, Writes}
 
-case class ForeignFhlEeaIncome(rentAmount: BigDecimal)
+case class ForeignFhlEeaIncome(rentAmount: Option[BigDecimal])
 
 object ForeignFhlEeaIncome {
-  implicit val format: OFormat[ForeignFhlEeaIncome] = Json.format[ForeignFhlEeaIncome]
+  implicit val writes: Writes[ForeignFhlEeaIncome] = Json.writes[ForeignFhlEeaIncome]
+  implicit val reads: Reads[ForeignFhlEeaIncome] =
+    (JsPath \ "rentAmount").readNullable[BigDecimal].map(rentAmount => ForeignFhlEeaIncome(rentAmount))
```

Comment:
now that I think about this; I'm not sure exactly what this achieves? tests still all pass when this is replaced with a regular reads